examples/front_ends/per_user_workflow/src/nat_per_user_workflow/register.py

```
@@ -0,0 +1,25 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Registration module for the per-user workflow example.
+
+This module imports all per-user functions and workflows to register them
+with NAT's plugin system.
+"""
+
+# Import to trigger registration
+from nat_per_user_workflow.per_user_functions import per_user_notepad  # noqa: F401
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)